src/server/routes/editor.js

```diff
@@ -245,7 +244,8 @@ router.get('/:id', (req, res, next) => {
 				markup,
 				page: 'profile',
 				props: escapeProps(props),
-				script: '/js/editor/editor.js'
+				script: '/js/editor/editor.js',
+				title: `Profile: ${props.editor.name}`
```

Comment:
I think I would prefer `${props.editor.name}'s Profile` instead, and same format for the other two editor routes.

Reply:
I also think writing title that way would be a better choice.